packages/cms/src/utils/formatters/abbreviate.js

```diff
@@ -1,4 +1,4 @@
-const libs = require("./libs");
+const libs = require("../libs");
```

Comment:
If this required correcting in `abbreviate.js` and `commas.js`, does it require fixing in other places? Could you please review all other functions in `/formatters` and adjust accordingly?